src/makeTxn.ts

```diff
@@ -35,7 +35,7 @@ import { RenameProperties, RenameProperty } from './types/utils';
  * @param suggestedParams - a dict holding common-to-all-txns args:
  * fee - integer fee per byte, in microAlgos. for a flat fee, set flatFee to true
  * flatFee - bool optionally set this to true to specify fee as microalgos-per-txn
- *      If the final calculated fee is lower than the protocol minimum fee, the fee will be increased to match the minimum.
+ *       If true, txn fee may fall below the ALGORAND_MIN_TX_FEE for the current network protocol
```

Comment:
I'm not sure how I feel about the `current network protocol` language here. I could be wrong, but I think wording in the SDKs is "timeless" for the most part, not referencing a specific network protocol, with the one exception being TEAL versioning. I'm not positive about this though – I'd welcome other opinions on this.

Reply:
In this case I agree we should remove the reference to the current protocol, saying "may fall below ALGORAND_MIN_TX_FEE" should be sufficient.